client/shared/src/settings/temporary/TemporarySettings.ts

```diff
@@ -52,6 +52,10 @@ export interface TemporarySettingsSchema {
     'search.results.collapseSmartSearch': boolean
     'search.input.recentSearches': RecentSearch[]
     'search.input.usedInlineHistory': boolean
+    // This is a temporary setting to allow users to easily switch
+    // between  having search  results be ranked or not. It's only
+    // used when the feature flag `"search-ranking": "true"` is set.
```

Comment:
ultra-nit on wording. The feature flag could be set to a rollout % (and not fixed to true).

```suggestion
    // This is a temporary setting to allow users to easily switch
    // between  having search  results be ranked or not. It's only
    // used when the feature flag `search-ranking` is enabled.
```